src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/config.groovy

```diff
@@ -8,7 +8,7 @@ f.entry(field: "gitHubAuthId", title:_("GitHub API credentials")) {
 f.entry(field: "adminlist", title: _("Admin list")) {
   f.textarea(default: descriptor.adminlist) 
 }
-f.entry(field: "useGitHubHooks", title: "Use github hooks for build triggering") {
+f.entry(field: "useGitHubHooks", title: "Use GitHub hooks for build triggering") {
```

Comment:
Changes like this should be broken out into a separate pull request.  Don't mix changes if you're proposing a large change.  This change isn't large but just general advice.